unlock-app/src/__tests__/hooks/useFiatKeyPrices.test.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
```

Comment:
Strange? Is this from a rebase?

Reply:
yes weird. I didnt touch anything there for sure

Reply:
Can you remove these files then? Just to make sure we're not overwriting something important?

Reply:
ok done